tests/baker_recipes.py

```diff
@@ -0,0 +1,24 @@
+from model_bakery.recipe import Recipe, related
+
+from api.models import EnrollmentTest, EnrollmentTestQuestion, EnrollmentTestQuestionOption
+
+QUESTIONS_AMOUNT = 4
+
+questions_options = [
+    Recipe(EnrollmentTestQuestionOption, text="Option 1"),
+    Recipe(EnrollmentTestQuestionOption, text="Option 2"),
+    Recipe(EnrollmentTestQuestionOption, text="Option 3"),
+    Recipe(EnrollmentTestQuestionOption, text="Option 4", is_correct=True),
+]
```

Comment:
IIRC, in the last iteration of owner's discussions, the test for adults had 4 options per question, the test for younger applicants had 3. I'm saying just in case, I see that you're just generating data for the test here :)

Reply:
I really don't know why I've made 4 options here. I wanted to show that we have some options and only one is correct.